domains/sarthak.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "dubey0613",
+           "email": "dubey.sarthak0613@gmail.com",
+           "discord": "877525618525364295"
+        },
+    
+        "record": {
+            "CNAME": "dubey0613.github.io/portfolio"
```

`CNAME` can't contain `/` 
```suggestion
            "CNAME": "dubey0613.github.io"
```

---

Do check again!